backend/composer/services/cs_ingestion/cs_ingestion_services.py

```diff
@@ -180,37 +180,33 @@ def get_or_create_sentence(statement: Dict) -> Tuple[Sentence, bool]:
 
 def create_or_update_connectivity_statement(statement: Dict, sentence: Sentence) -> Tuple[ConnectivityStatement, bool]:
     reference_uri = statement[ID]
-
-    connectivity_statement, created = ConnectivityStatement.objects.update_or_create(
-        reference_uri__exact=reference_uri,
-        defaults={
-            "sentence": sentence,
-            "knowledge_statement": statement[LABEL],
-            "sex": get_sex(statement),
-            "circuit_type": get_circuit_type(statement),
-            "functional_circuit_role": get_functional_circuit_role(statement),
-            "phenotype": get_phenotype(statement),
-            "projection_phenotype": get_projection_phenotype(statement),
-            "reference_uri": statement[ID],
-        }
-    )
+    defaults = {
+        "sentence": sentence,
+        "knowledge_statement": statement[LABEL],
+        "sex": get_sex(statement),
+        "circuit_type": get_circuit_type(statement),
+        "functional_circuit_role": get_functional_circuit_role(statement),
+        "phenotype": get_phenotype(statement),
+        "projection_phenotype": get_projection_phenotype(statement),
+        "reference_uri": statement[ID],
+        "state": CSState.EXPORTED,
+    }
+
+    if ConnectivityStatement.objects.filter(reference_uri__exact=reference_uri).exists():
```

Comment:
@afonsobspinto suggestion: did you consider using `.get_or_create` (see https://docs.djangoproject.com/en/5.0/ref/models/querysets/#get-or-create) ?